File: Src/Icm.ContextConsole.Example/StringContext.cs

using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using Icm.ContextConsole;

public class StringContext : BaseContext
{

	public void Concat()
	{
		dynamic num1 = Interactor.AskString("String 1");
		dynamic num2 = Interactor.AskString("String 2");

		Interactor.ShowMessage(string.Format("{0} + {1} = {2}", num1, num2, num1 + num2));
	}

}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================

File: Src2/Icm.ContextConsole.Example/StringContext.vb
Imports Icm.ContextConsole

Public Class StringContext
    Inherits BaseContext

    Public Sub Concat()
        Dim num1 = Interactor.AskString("String 1")
        Dim num2 = Interactor.AskString("String 2")

        Interactor.ShowMessage(String.Format("{0} + {1} = {2}", num1, num2, num1 & num2))
    End Sub

End Class